modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/db/Statement.scala

```diff
@@ -53,8 +54,8 @@ object Statement {
   case class FoldersCopy(source: S3.Folder) extends Statement
 
   // Loading
-  case class EventsCopy(path: S3.Folder, compression: Compression, columns: List[String]) extends Statement with Loading {
-    def table: String = "events"
+  case class EventsCopy(path: S3.Folder, compression: Compression, columnsToCopy: List[String], columnsToSkip: List[String]) extends Statement with Loading {
```

Comment:
Similar case as in `getLoadStatements` and columns - some case class/alias? With 2 params like `List[String]` I have a feeling it's easy to mix them up! :)